assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.stories.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * DashboardUniqueVisitorsWidget Component Stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { CORE_WIDGETS, WIDGET_WIDTHS, WIDGET_AREA_STYLES } from '../../../../googlesitekit/widgets/datastore/constants';
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { accountsPropertiesProfiles } from '../../datastore/__fixtures__';
+import { STORE_NAME } from '../../datastore/constants';
+import { provideModules, provideSiteInfo } from '../../../../../../tests/js/utils';
+import { getAnalyticsMockResponse } from '../../util/data-mock';
+import WidgetAreaRenderer from '../../../../googlesitekit/widgets/components/WidgetAreaRenderer';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
+import DashboardUniqueVisitorsWidget from './DashboardUniqueVisitorsWidget';
+
+const areaName = 'moduleAnalyticsMain';
+const widgetSlug = 'analyticsUniqueVisitors';
+const currentEntityURL = 'https://www.example.com/example-page/';
+const options = {
+	startDate: '2020-08-11',
+	endDate: '2020-09-07',
+	dimensionFilters: { 'ga:channelGrouping': 'Organic Search' },
+	dimensions: [ 'ga:date', 'ga:channelGrouping' ],
+	metrics: [
+		{
+			expression: 'ga:users',
+			alias: 'Users',
+		},
+	],
+};
+const optionsEntityURL = {
+	...options,
+	url: currentEntityURL,
+};
+const optionsCompare = {
+	compareStartDate: '2020-07-14',
+	compareEndDate: '2020-08-10',
+	startDate: '2020-08-11',
+	endDate: '2020-09-07',
+	dimensionFilters: { 'ga:channelGrouping': 'Organic Search' },
+	dimensions: [ 'ga:channelGrouping' ],
+	metrics: [
+		{
+			expression: 'ga:users',
+			alias: 'Total Users',
+		},
+	],
+};
+const optionsCompareEntityURL = {
+	...optionsCompare,
+	url: currentEntityURL,
+};
```

Comment:
This looks like it would be too easy for the story to get out of sync with the component like this. Maybe we can expose the base/default options from the component as a static property or export? The dates should be handled automatically though by setting the reference date. Same with the entity URL.

We'd done something like this before in a similar situation by exposing select functions on the component but it ended up being replaced with widget state. In this case, maybe we can expose functions for selecting the report args which would be used in the component as well. 🤔 

I will see about trying this on another branch.